package.json

```diff
@@ -986,7 +986,7 @@
     "@folio/stripes-core": "^10.2.0",
     "@folio/stripes-inventory-components": "^1.0.0",
     "@folio/stripes-marc-components": "^1.0.1",
-    "@folio/stripes-smart-components": "^9.2.0",
+    "@folio/stripes-smart-components": "^10.0.0",
```

Comment:
We can't just bump `stripes-smart-components` to a new version. Since we take it from `stripes` - it has to be a new stripes release with a new major version. Let's wait on Zak's comment in STSMACOM PR